src/common/badge.tsx

```diff
@@ -1,13 +1,9 @@
-const Badge = ({
-  icon,
-  className
-}: {
-  icon: string
-  className: string
-}): JSX.Element => (
-  <div className={className}>
-    <img src={icon} alt="" />
-  </div>
+import { BadgeProps } from '../types/interfaces'
```

Comment:
leave props interfaces inside of component or in separate file in the same folder